core/src/smtp/parser.rs

```diff
@@ -95,6 +95,10 @@ pub fn is_invalid(e: &str, email: &EmailAddress) -> bool {
 	|| e.contains("address error")
 	// E-mail address is not handled by this system (on @kaimayfair.co.uk)
 	|| e.contains("address is not handled")
+	// permanent: 5.1.1 recipient is not exist (on @sim.com)
+	|| e.contains("recipient is not exist")
+	// permanent: 5.1.1 <EMAIL>  Recipient not found. (on @4polymer.com)
+	|| e.contains("Recipient not found")
```

Comment:
```suggestion
	|| e.contains("recipient not found")
```

I lowercase the error message before checking this `contains`